mgr/src/cmds/nodes.cr

```diff
@@ -88,7 +94,8 @@ handler "node.remove" do |args|
   next unless (args.script_mode || yes("Are you sure you want to remove the Node from the Pool?"))
 
   api_call(args, "Failed to Remove the Node") do |client|
-    client.pool(args.pool_name).node(node_name).delete
+    node = client.pool(args.pool_name).node(node_name).delete
+    handle_json_output(node, args)
```

Comment:
Remove `node` var and use `handle_json_output(nil, args)`

Reply:
Ack to all responses on passing of `nil` for empty responses.